cosmos/sql/tools/java/ycsb/recipes/scan/README.md

```diff
@@ -0,0 +1,15 @@
+## Overview
+The query used in the scan operation performs a [cross-partition query](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-query-container#cross-partition-query). In Cosmos DB, when you run a cross-partition query on a container, you are effectively running one query per physical partition. As the partitions increase, both the latency and RUs consumed increase significantly. Cosmos DB SDKs can be tuned to [parallelize cross-partition queries](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/how-to-query-container#parallel-cross-partition-query) to improve the latency, but the RU charges will remain high. The following table compares the latencies for parallel and serial query execution and the effect of physical partition on latency. 
```

Comment:
The query used in the scan -> The query used in the YCSB scan ?

Reply:
How about "Having a limited number of cross-partition queries is unavoidable in some data models but as a best practice it’s highly recommended to design data models to avoid cross-partition queries, especially for large containers"

Reply:
Also pointed to the official YCSB repo